### PR TITLE
Introduce server settings side panel item

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- Issue `#1` or (None)

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/gentle-deers-stare.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": minor
+"@wso2is/i18n": patch
```
Comment (Achintha444) [inline .changeset/gentle-deers-stare.md:3]: Bump myaccount and identity-apps-core

File: apps/console/src/features/server/pages/server.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    overflow: hidden;
+    height: 40px;
+    width: 40px;
+    display: flex;
+    -webkit-box-align: center;
+    align-items: center;
+    -webkit-box-pack: center;
+    justify-content: center;
+    flex: 0 0 auto;
+}
```
Comment (Achintha444) [inline apps/console/src/features/server/pages/server.scss:50]:
```suggestion
}

```

Comment (DonOmalVindula) [reply apps/console/src/features/server/pages/server.scss]: Fixed with 466fe513fbca5ed9dc13f15d2633888895fb948b

File: .changeset/gentle-deers-stare.md (hunk 1 above)
Comment (DonOmalVindula) [reply .changeset/gentle-deers-stare.md:3]: Fixed with 466fe513fbca5ed9dc13f15d2633888895fb948b

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**